src/main/java/base/DBService.java

```diff
@@ -0,0 +1,11 @@
+package base;
+
+import dbService.DBException;
+
+public interface DBService {
+    UserProfile getUser(String login) throws DBException;
+
+    void create() throws DBException;
+
+    long addUser(UserProfile userProfile) throws DBException;
```

Comment:
better use wrapper here

Reply:
You are right. We must use all wrappers, as I known.